maquetado/scss/details.scss

```diff
@@ -99,6 +99,11 @@
     right: 40px;
     bottom: -3px;
   }
+
+  &:hover:before {
+    animation-name: redirect-rotate;
+    animation-duration: 1s;
```

Comment:
Really nice animation!! 

You can use a shorthand for this though -> animation: name duration;